plasmapy/formulary/tests/test_ionization.py

```diff
@@ -3,17 +3,37 @@
 import pytest
 from astropy.constants import c
 
-from plasmapy.formulary.ionization import Z_bal
+from plasmapy.formulary.ionization import Saha, ionization_balance
 
 
-def test_Z_bal():
+def test_ionization_balance():
     n = 1e19 * u.m ** -3
     T_e = 5000 * u.K
 
-    assert (Z_bal(n, T_e) * u.dimensionless_unscaled).unit == u.dimensionless_unscaled
+    assert (
+        ionization_balance(n, T_e) * u.dimensionless_unscaled
+    ).unit == u.dimensionless_unscaled
 
     with pytest.warns(u.UnitsWarning):
-        Z_bal(1e19, T_e)
+        ionization_balance(1e19, T_e)
 
     with pytest.raises(u.UnitTypeError):
-        Z_bal(2e19 * u.kg, T_e)
+        ionization_balance(2e19 * u.kg, T_e)
+
+
+def test_Saha():
+    g_j = 2
+    g_k = 1
+    n_e = 1e19 * u.m ** -3
+    E_jk = 10 * u.Ry
+    T_e = 5000 * u.K
+
+    assert (
+        Saha(g_j, g_k, n_e, E_jk, T_e) * u.dimensionless_unscaled
+    ).unit == u.dimensionless_unscaled
```

Comment:
```suggestion
    val = Saha(g_j, g_k, n_e, E_jk, T_e)
    assert val.unit == u.dimensionless_unscaled
    assert np.isclose(val.value, 2.4775608756800435e-129)
```

Reply:
@KhalilBryant don't forget to update this test.

Reply:
@KhalilBryant I think this is the last bit and then we can merge, unless you have other changes you'd like to do.